### ISSUE TITLE
whecho: error: unrecognized arguments

### ISSUE THREAD

**Issue by wndyy** [cvraut/whecho#5]
Used whecho command follow by argument "let's see if this works" and got an error.

```
whecho let's see if this works
usage: whecho [-h] [--version] [-m MSG] [--init] [-u URL] [-d] [message]
whecho: error: unrecognized arguments: see if this works
```

I expected to see the message "let's see if this works" in my server.

 - OS: Windows 11
 - Webhook client: Discord
 - Python Version 3.6

**Comment by cvraut** [comment]
whecho only takes one positional argument for now, surround your request with quotes like

`whecho "let's see if this works"`

**Comment by cvraut** [comment]
just released version 0.0.4

I updated whecho to allow for variable number of positional arguments. Try it now & lmk if it works

**Comment by wndyy** [comment]
works as expected. thanks.
